css/style.css

```diff
@@ -0,0 +1,187 @@
+* {
+  box-sizing: border-box;
+}
+
+h1, h2, p, body, img, div {
+  margin: 0;
+  padding: 0;
+  font-family: 'Source Code Pro', monospace;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background: url('../img/bg.png') no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.blink-text {
+  animation: blinker 2s linear infinite;
+}
+
+@keyframes blinker {
+  50% {
+    opacity: 0;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  padding-top: 35vh;
+}
+
+.header h1 {
+  font-size: 10vw;
+  color: #FFFFFF;
+  text-shadow: 5px 5px #00000080;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  margin-top: 10vh;
+}
+
+.nav button {
+  display: flex;
+  background-color: #FFFFFF00;
+  color: #FFFFFF;
+  border: 2px solid #FFFFFF;
+  border-radius: 5px;
+  width: 30%;
+  height: 3rem;
+  margin: 5vw;
+  align-items: center;
+  justify-content: center;
+  box-shadow: 5px 5px #00000080;
+}
+
+.nav button:hover {
+  background-color: #FFFFFF;
+  color: #000000;
+}
+
+.about {
+  margin: 5vw;
+  color: #FFFFFF;
+}
+
+.about h2 {
+  margin-top: 40vh;
+  margin-bottom: 1vh;
+}
+
+.about p {
+  margin-bottom: 10vh;
+}
+
+.about-flexbox {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-flexbox img {
+  display: flex;
+  height: 28rem;
+  width: 28rem;
+  border-radius: 40px;
+  box-shadow: 5px 5px #00000080;
+}
+
+.projects h2 {
+  margin: 25vw 5vw 1vw 5vw;
+  color: #FFFFFF;
+}
+
+.project-container {
+  margin-left: 11.5vw;
+  margin-right: 11.5vw;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.project-container img {
+  padding: 0 2vw;
+  text-align: center;
+}
+
+.project-container div {
+  flex: 0 0 33%;
+  margin-bottom: 2%;
+  display: flex;
+}
+
+.button-overlay {
+  position: absolute;
+  z-index: 9;
+  margin-top: 260px;
+  margin-left: 7rem;
+}
+
+.button-overlay button {
+  background-color: #FFFFFFCC;
+  color: #000000;
+  border: 1px solid #555555;
+  border-radius: 5px;
+}
+
+.button-overlay button:hover {
+  background-color: #55555580;
+  color: #FFFFFF;
+}
+
+#upbutton {
+  display: none; /* Hidden by default */
+  position: fixed; /* Fixed/sticky position */
+  bottom: 20px; /* Place the button at the bottom of the page */
+  right: 30px; /* Place the button 30px from the right */
+  z-index: 99; /* Make sure it does not overlap */
+  outline: none; /* Remove outline */
+  padding: 15px; /* Some padding */
```

Comment:
Really like the commenting here as it's a lot of CSS for the top button function you have.